runtime/moonbase/src/lib.rs

```diff
@@ -522,6 +522,41 @@ impl pallet_treasury::Config for Runtime {
 	type SpendFunds = ();
 }
 
+parameter_types! {
+	pub const BasicDeposit: Balance = currency::deposit(1, 246);
+	pub const FieldDeposit: Balance = currency::deposit(0, 66);
+	pub const SubAccountDeposit: Balance = currency::deposit(1, 41);
+	pub const MaxSubAccounts: u32 = 100;
+	pub const MaxAdditionalFields: u32 = 100;
+	pub const MaxRegistrars: u32 = 20;
```

Comment:
can you explain the deposit constants? maybe add comments like in the Kusama runtime:
```
parameter_types! {
	// Minimum 100 bytes/KSM deposited (1 CENT/byte)
	pub const BasicDeposit: Balance = 1000 * CENTS;       // 258 bytes on-chain
	pub const FieldDeposit: Balance = 250 * CENTS;        // 66 bytes on-chain
	pub const SubAccountDeposit: Balance = 200 * CENTS;   // 53 bytes on-chain
	pub const MaxSubAccounts: u32 = 100;
	pub const MaxAdditionalFields: u32 = 100;
	pub const MaxRegistrars: u32 = 20;
}
```

Reply:
Done